CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/RunPane.java

```diff
@@ -173,31 +176,37 @@ public RunPane(final ImportController importController, final String displayText
         filterField = new TextField();
         filterField.setFocusTraversable(false);
         filterField.setMinHeight(USE_PREF_SIZE);
-        filterField.setText("Filter");
-        filterField.setStyle("-fx-background-color: black; -fx-text-fill: white;");
+        filterField.setPromptText("Start typing to search, e.g.:first_name==\"NICK\"");
+        filterField.setStyle("-fx-background-color: black; -fx-text-fill: white;-fx-prompt-text-fill:white;");
```

Comment:
a minor thing but is it possible to change the prompt text colour to grey? Just to give enough distinction between the prompt text and user input. Looks good otherwise